features/projects/components/project-list/project-list.tsx

```diff
@@ -27,7 +39,17 @@ export function ProjectList() {
 
   if (isError) {
     console.error(error);
-    return <div>Error: {error.message}</div>;
+    return (
+      <Badge color={BadgeColor.error} size={BadgeSize.lg}>
+        <RiErrorWarningLine style={{ padding: 3 }} />{" "}
```

Comment:
You can export the icons from the Figma design. No need to use an external package 😉